index.html

```diff
@@ -16,11 +16,15 @@
           <div class="wrapper">
             <nav>
               <ul>
-                <li class="name"><span>pius ssekweyama</span><i class="fa fa-bars mobile-menu" aria-hidden="true"></i></li>
-                <li class="desktop-menu">portfolio</li>
-                <li class="desktop-menu">about</li>
-                <li class="desktop-menu">contact</li>
-                <li class="social desktop-menu">
+                <li class="name">
+                  <span>pius ssekweyama</span>
+                  <i class="fa fa-bars mobile-menu" onclick="toggleMenu()" aria-hidden="true"></i>
+                  <i class="fa fa-times mobile-menu menu" onclick="toggleMenu()" aria-hidden="true"></i>
+                </li>
+                <li class="toggle menu"><a href="#portfolio">portfolio</a></li>
+                <li class="toggle menu"><a href="#about">about</a></li>
+                <li class="toggle menu"><a href="#contact">contact</a></li>
+                <li class="social toggle menu">
                   <i class="fa fa-envelope" aria-hidden="true"></i>
```

Comment:
- [ ] Please make sure that your design matches with the one provided on Figma, (e.g., font, colors, images, text, margins).

Your design  | Figma design
------------- | -------------
<img width="254" alt="Screenshot 2022-10-17 at 17 26 58" src="https://user-images.githubusercontent.com/77038610/196203445-b7f59fea-2e09-4f71-a50b-583b09686d54.png">  | <img width="260" alt="Screenshot 2022-10-17 at 17 27 27" src="https://user-images.githubusercontent.com/77038610/196203590-5df5f41a-85d7-4537-ad17-076ae34ac790.png">